app/Resources/translations/default/AdminNotificationsError.xlf

```diff
@@ -1147,17 +1198,22 @@ Comment: check if some ids are in list_skip_actions and forbid deletion</note>
       <trans-unit id="505bbc38260c669d0793757e414caecc">
         <source>The %s field is not valid</source>
         <target>The %s field is not valid</target>
-        <note>Line: 493</note>
+        <note>Line: 572</note>
       </trans-unit>
       <trans-unit id="777e4c7c4a3cc47eb332a7a4022c983e">
         <source>Unexpected error occurred.</source>
         <target>Unexpected error occurred.</target>
-        <note>Line: 416</note>
+        <note>Line: 427</note>
       </trans-unit>
       <trans-unit id="b3f69fdbd12765a4667492d013ea3d81">
         <source>You need permission to edit this.</source>
         <target>You need permission to edit this.</target>
-        <note>Line: 401</note>
+        <note>Line: 412</note>
+      </trans-unit>
+      <trans-unit id="4ea19e936f253180edaf4cff86919606">
+        <source>You must select at least one element to perform a bulk action.</source>
```

Comment:
Indeed `You need to select at least one item to use the bulk action.` was used in a legacy controller (only one instance) and `You must select at least one element to perform a bulk action.` in the more recent Currency controller. Which one should we use in the future?

Reply:
Well, I had kept the first one because we tend to use "item" more. In that case, would it be possible to use "You must select at least **one item** to perform a bulk action." ?